code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -958,5 +958,74 @@
 /datum/movespeed_modifier/careful_driving
 	multiplicative_slowdown = 3
 
+/datum/status_effect/midas_blight
+	id = "midas_blight"
+	alert_type = /atom/movable/screen/alert/status_effect/midas_blight
+	status_type = STATUS_EFFECT_REPLACE
+	remove_on_fullheal = TRUE
+	processing_speed = STATUS_EFFECT_FAST_PROCESS
```

Comment:
This is the default processing speed I believe, so this is redundant
```suggestion
```

Reply:
it was still only processing every second instead of 0.2 seconds, so I thought it was broken

Reply:
I FIGURED OUT `tick_interval` IS A VARIABLE HOLD ON LET ME TRY THAT